<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wifimanager</class>
 <widget class="QMainWindow" name="wifimanager">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>614</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>wifimanager</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>1171</width>
      <height>561</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>WIFI menu</string>
     </attribute>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>30</y>
        <width>491</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>160</width>
        <height>195</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="refrech">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>scan</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../Downloads/refresh.png</normaloff>../../../Downloads/refresh.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connect">
         <property name="text">
          <string>connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnect">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="forgot">
         <property name="text">
          <string>forget</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="pass">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>220</y>
        <width>251</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Enter password</string>
      </property>
     </widget>
     <widget class="QLabel" name="statuslabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>991</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="showpass">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>220</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>show password</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>add Wifi</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>391</width>
        <height>35</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>SSID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ssid2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter Network name</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="Security">
         <property name="text">
          <string>Security:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listSecurity">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>23</height>
          </size>
         </property>
         <property name="movement">
          <enum>QListView::Movement::Static</enum>
         </property>
         <property name="isWrapping" stdset="0">
          <bool>false</bool>
         </property>
         <property name="resizeMode">
          <enum>QListView::ResizeMode::Fixed</enum>
         </property>
         <property name="layoutMode">
          <enum>QListView::LayoutMode::SinglePass</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::ViewMode::ListMode</enum>
         </property>
         <property name="selectionRectVisible">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>none</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WPA/WPA2-Personal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WPA3-Personal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WPA/WPA2-Enterprise</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WPA3-Enterprise</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>59</x>
        <y>90</y>
        <width>391</width>
        <height>35</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="pass2">
         <property name="text">
          <string>password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pass2Line">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="Eaplabel">
         <property name="text">
          <string>EAP method:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="Eap">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>PEAP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TTLS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TLS</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="check2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>show password</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>391</width>
        <height>43</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="idenity">
         <property name="text">
          <string>identity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="idenityLine"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="addnet">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>390</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Network</string>
      </property>
     </widget>
     <widget class="QLabel" name="addstatus">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>621</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="hiddenbox">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>hidden</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>About us</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>1001</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Designed by : Zaid benkhaira    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Coded By Oualid &amp; Zaid Benkhaira</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Optimized By : Marouane Drioueche</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>0</y>
        <width>1001</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Source Code Pro Medium</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>WifiManager version : 1.0 (BETA)!!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>               Welcome to Wifi_Manger nmcli based</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
